### PR TITLE
adding release notes for 0.17.x

### LINKED ISSUE #9579: 0.17.x release notes
Generated 06/17/2017
 - Incorrect hostname shown in titlemode when host is very long. ([#9500](https://github.com/brave/browser-laptop/issues/9500))
 - Upgrade to Chromium 59.0.3071.104. ([#9499](https://github.com/brave/browser-laptop/issues/9499))
 - Middle click on pinned tab causes all opened tabs to show as pinned tabs. ([#9492](https://github.com/brave/browser-laptop/issues/9492))
 - Clicking menu bar items in 0.16.x and 0.17.1 does nothing. ([#9470](https://github.com/brave/browser-laptop/issues/9470))
 - Tab title isn't hidden for session tabs after the tab threshold is reached. ([#9466](https://github.com/brave/browser-laptop/issues/9466))
 - AppState.sites sort takes a long time (Every navigation +200 ms [w/ 10K bookmarks]). ([#9427](https://github.com/brave/browser-laptop/issues/9427))
 - "Select its parent tab" setting not working as expected. ([#9395](https://github.com/brave/browser-laptop/issues/9395))
 - Bookmark title is not selected automatically. ([#9091](https://github.com/brave/browser-laptop/issues/9091))
 - Improve RPM packaging. ([#9074](https://github.com/brave/browser-laptop/issues/9074))
 - Button not disabled on add new bookmark modal by default. ([#9019](https://github.com/brave/browser-laptop/issues/9019))
 - Add compact bravery panel with Aphrodite. ([#8954](https://github.com/brave/browser-laptop/issues/8954))
 - CSP referrer errors. ([#8920](https://github.com/brave/browser-laptop/issues/8920))
 - Build hangs in tmux on macOS. ([#8916](https://github.com/brave/browser-laptop/issues/8916))
 - Importing bookmarks after sync causes loss of hierarchy. ([#8892](https://github.com/brave/browser-laptop/issues/8892))
 - Show all button overlaps on ledger table entry. ([#8869](https://github.com/brave/browser-laptop/issues/8869))
 - Allow <site> to <action>?. ([#8826](https://github.com/brave/browser-laptop/issues/8826))
 - Noscript approvals from private tabs apply to regular tabs. ([#8779](https://github.com/brave/browser-laptop/issues/8779))
 - Saving a document doesn't show correct origin. ([#8698](https://github.com/brave/browser-laptop/issues/8698))
 - Pinned tab notification doesn't have the caret and shown across all tabs. ([#8626](https://github.com/brave/browser-laptop/issues/8626))
 - Unable to view HBOGO video. ([#8581](https://github.com/brave/browser-laptop/issues/8581))
 - Import data gets duplicated after sync. ([#8508](https://github.com/brave/browser-laptop/issues/8508))
 - Offer option to have URL bar fill empty space ("wide URL bar"). ([#8421](https://github.com/brave/browser-laptop/issues/8421))
 - Extension: Honey. ([#8118](https://github.com/brave/browser-laptop/issues/8118))
 - Unhide bitwarden password manager. ([#7921](https://github.com/brave/browser-laptop/issues/7921))
 - Integrate the new welcome screen. ([#7821](https://github.com/brave/browser-laptop/issues/7821))
 - Tab preview on hover doesn't work after closing tab until moving the mouse out of tabs bar. ([#7606](https://github.com/brave/browser-laptop/issues/7606))
 - 50-80ms delay when closing a tab. ([#7395](https://github.com/brave/browser-laptop/issues/7395))
 - Right clicking tab being previewed causes focus to change. ([#7327](https://github.com/brave/browser-laptop/issues/7327))
 - Browser performance reduces when large amount of bookmarks are imported. ([#7240](https://github.com/brave/browser-laptop/issues/7240))
 - Clicking on disabled Ad/Tracker count enables the <li> element under braveryPanelBody. ([#7026](https://github.com/brave/browser-laptop/issues/7026))
 - Autofill form option blinks and cannot be clicked. ([#5438](https://github.com/brave/browser-laptop/issues/5438))
 - Brave Browser very slow to open or close tabs. ([#4848](https://github.com/brave/browser-laptop/issues/4848))
 - Sort adblock filter lists alphabetically. ([#4501](https://github.com/brave/browser-laptop/issues/4501))
 - CSS for visited links ( A:Visited ) does not work. ([#512](https://github.com/brave/browser-laptop/issues/512))

### PR DESCRIPTION
**Pull request by alexwykoff**
Auditors : @bbondy, @luixxiul, @bsclifton

Fixes #9579

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: tools/publish_release.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def release_channel():
   return channel
 
 def publish_release(github, release_id):
-  data = dict(draft=False)
+  data = dict(draft=False, prerelease=True)
```
Comment (luixxiul) [inline tools/publish_release.py]: is this change intended?

Comment (alexwykoff) [inline tools/publish_release.py]: Oh funny, not that I'm aware of @luixxiul . I based off of master, so I'm not sure why that deviated. 

@bsclifton should we default to prerelease like that?

**Comment by alexwykoff** [comment]
@bsclifton your changes lgtm 👍

File: tools/publish_release.py (hunk 1 above)
Comment (bsclifton) [inline tools/publish_release.py]: oh crap- sorry this must have got added when I did a `git commit -a`; backing this out...

**Comment by bsclifton** [comment]
@alexwykoff awesome 😄  I just backed out the unintended change to `tools\publish_release.py` (thanks for catching @luixxiul!)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+- Fixed bookmark title is not selected automatically. ([#9091](https://github.com/brave/browser-laptop/issues/9091))
+- Fixed improve RPM packaging. ([#9074](https://github.com/brave/browser-laptop/issues/9074))
+- Fixed button not disabled on add new bookmark modal by default. ([#9019](https://github.com/brave/browser-laptop/issues/9019))
+- Fixed show all button overlaps on ledger table entry. ([#8869](https://github.com/brave/browser-laptop/issues/8869))
+- Fixed pinned tab notification doesn't have the caret and shown across all tabs. ([#8626](https://github.com/brave/browser-laptop/issues/8626))
+- Fixed viewing HBOGO video. ([#8581](https://github.com/brave/browser-laptop/issues/8581))
+- Fixed Allow notification bar choices. ([#8826](https://github.com/brave/browser-laptop/issues/8826))
+- Fixed tab preview on hover. ([#7606](https://github.com/brave/browser-laptop/issues/7606))
+- Fixed right clicking tab being previewed causes focus to change. ([#7327](https://github.com/brave/browser-laptop/issues/7327))
+- Fixed clicking on disabled Ad/Tracker count enables the <li> element under braveryPanelBody. ([#7026](https://github.com/brave/browser-laptop/issues/7026))
```
Comment (luixxiul) [inline CHANGELOG.md]: `<li>` must be escaped.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,45 @@
...
+- Fixed autofill form option blinks and cannot be clicked. ([#5438](https://github.com/brave/browser-laptop/issues/5438))
+- Fixed build hangs in tmux on macOS. ([#8916](https://github.com/brave/browser-laptop/issues/8916))
+- Upgrade Muon to 4.1.4. ([#9645])(https://github.com/brave/browser-laptop/issues/9645))
```
Comment (luixxiul) [inline CHANGELOG.md]: ` ([#9645](https://github.com/brave/browser-laptop/issues/9645))`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,45 @@
 # Changelog
 
+## [0.17.7](https://github.com/brave/browser-laptop/releases/tag/v0.17.7dev)
+- Added a new "Welcome" screen. ([#7821](https://github.com/brave/browser-laptop/issues/7821))
```
Comment (bbondy) [inline CHANGELOG.md]: I'd move this down since we aren't highlighting or showing it on first run in this release.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,45 @@
...
+- Fixed tab title isn't hidden for session tabs after the tab threshold is reached. ([#9466](https://github.com/brave/browser-laptop/issues/9466))
+- Fixed "Select its parent tab" setting not working as expected. ([#9395](https://github.com/brave/browser-laptop/issues/9395))
+- Fixed closing tabs results in white page. ([#9306](https://github.com/brave/browser-laptop/issues/9306))
+- Fixed bookmark title is not selected automatically. ([#9091](https://github.com/brave/browser-laptop/issues/9091))
+- Fixed improve RPM packaging. ([#9074](https://github.com/brave/browser-laptop/issues/9074))
+- Fixed button not disabled on add new bookmark modal by default. ([#9019](https://github.com/brave/browser-laptop/issues/9019))
+- Fixed show all button overlaps on ledger table entry. ([#8869](https://github.com/brave/browser-laptop/issues/8869))
+- Fixed pinned tab notification doesn't have the caret and shown across all tabs. ([#8626](https://github.com/brave/browser-laptop/issues/8626))
+- Fixed viewing HBOGO video. ([#8581](https://github.com/brave/browser-laptop/issues/8581))
```
Comment (bbondy) [inline CHANGELOG.md]: HBO GO

File: CHANGELOG.md (hunk 2 above)
Comment (bbondy) [inline CHANGELOG.md]: to:
```
`<li>`
```
